### PR TITLE
Update getting-started doc post dao launch

### PR DESCRIPTION
**Pull request by m52go**
Long-overdue post-v1.0 update. Main items were mentioning BSQ and updating screenshots, but this PR includes the following too:
- update security deposit recommendation, as it's now a percentage (not fixed 0.01 BTC)
- add note on backups to "Next steps" section
- add mediators to mention of arbitrators
- update FAQs links

Addresses #135.

### CODE REVIEW

**Comment by m52go** [comment]
@erciccione @wiz would one of you mind reviewing this? If so, just announce your intention to review here first so the other doesn't unknowingly duplicate efforts.

File: getting-started.adoc [hunk 1]
```diff
@@ -81,11 +79,13 @@ image::after-adding-fiat-account.png[Configuring a national currency account]
...
 
-If you don't already own this much bitcoin, you'll need to obtain it. How? You've got options. Friends and family are often the best option—simply buy a small amount of bitcoin directly from someone you trust. Otherwise, you can try a https://coinatmradar.com/[local bitcoin ATM] (many don't require ID for small transactions), https://www.meetup.com/topics/bitcoin/[bitcoin meetups/events], or vouchers.
+As of this writing, you would need 0.0012 BTC to take an offer to buy 0.01 BTC that requires a 10% security deposit. But please note that mining fees can fluctuate, some sellers may require bigger security deposits, and you'll need more for bigger trades, so it may be better to acquire more than the bare minimum so you have more flexibility.
+
+If you don't already own the bitcoin you need, you'll need to obtain it. How? You've got options. Friends and family are often the best option—simply buy a small amount of bitcoin directly from someone you trust. Otherwise, you can try a https://coinatmradar.com/[local bitcoin ATM] (many don't require ID for small transactions), https://www.meetup.com/topics/bitcoin/[bitcoin meetups/events], or vouchers.
```
Comment (erciccione) [inline getting-started.adoc]: `Friends and family are often the best option—simply buy a small amount of bitcoin directly from someone you trust.` This sounds a bit unrealistic at the present state of the marker IMO (How many of us have family members who actively buy and sell BTC?). Maybe better to substute this text with something more generic, omitting *Friends and family are often the best option*.

File: getting-started.adoc [hunk 2]
```diff
@@ -150,9 +150,32 @@ Bisq will tell you the address of the trade wallet and exactly how much to send
...
+
+What's this about? Is Bisq shilling its own altcoin?
+
+Hold your horses!
+
+BSQ is **colored bitcoin** that funds and governs Bisq. Remember, Bisq is a _network_ run by _software_—not a _company_ run by a _CEO_—but decisions still need to be made and developers still need to be paid. BSQ colored bitcoin and the Bisq DAO achieve these functions for the Bisq network without a corporate entity, legal jurisdiction, or any designated "leadership".
+
+There was never an ICO, just <<dao/phase-zero#, extensive conception>>, development, and testing over 4+ years to make it a reality.
+
+**Using BSQ is optional**. But buying BSQ pays Bisq's developers and makes the project sustainable, so it's preferred, and this is reflected in trading fees—BSQ fees are considerably lower than BTC fees.
```
Comment (erciccione) [inline getting-started.adoc]: s/preferred/encouraged

File: getting-started.adoc (hunk 1 above)
Comment (m52go) [reply getting-started.adoc]: Friends/family is merely a suggestion. Appreciate the thought, but will leave it in since it was already there.

**Pull request merged**